test/package.json

```diff
@@ -1,11 +1,10 @@
 {
   "dependencies": {
+    "chalk": "^4.1.2",
+    "rollup": "4.22.0",
     "tslib": "file:..",
```

Comment:
Maybe not new in this PR but IIRC this tells the package manager to copy the package and not make a link, making iterative testing more annoying.

Reply:
It’s definitely a symlink in npm